docs/usage.rst

```diff
@@ -90,7 +95,7 @@ Some operations can be paginated::
 The pagination data is available in the ``page`` property::
 
    >>> response.page
-   {'total_results': 52, 'page_size': 2, 'page_number': 1, 'total_pages': 26}
+   {'total_results': 52, 'page_size': 2, 'page': 1, 'total_pages': 26}
```

Comment:
Are you sure? I think it's `page_number`.

Reply:
Defined here right?
https://github.com/fedora-infra/fasjson/blob/master/fasjson/web/utils/pagination.py#L9

Reply:
Ah, no, that's the name of the argument you must use to **request** a page, tha's not what you get in the **page** data dictionary. This one is documented at line 19 of that same file.

But I agree it's confusing, I'll rename the argument in fasjson to `page_number` too.